src/middlewared/middlewared/plugins/ldap.py

```diff
@@ -442,13 +469,27 @@ class Config:
                 "ldap_update.hostname",
                 "The LDAP hostname parameter is required."
             )
+        for idx, uri in enumerate(new["uri_list"]):
+            parsed = urlparse(uri)
+            try:
+                port = parsed.port
+
+            except ValueError:
+                verrors.add(f"ldap_update.hostname.{idx}",
+                            "Port number [{port}] is not an integer.")
```

Comment:
Better to say "Invalid port number" because
```
>>> urllib.parse.urlparse('url://localhost:65536').port
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python3.6/urllib/parse.py", line 171, in port
    raise ValueError("Port out of range 0-65535")
ValueError: Port out of range 0-65535
```